-1

Tested at: 2f4d290dfc6f14732b152611906115290da71d6a

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-23310/28160/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling  /build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_9_3_X_2018-05-20-0000/src/SimG4Core/CustomPhysics/src/CustomParticleFactory.cc 
>> Compiling  /build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_9_3_X_2018-05-20-0000/src/SimG4Core/CustomPhysics/src/DummyChargeFlipProcess.cc 
>> Compiling  /build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_9_3_X_2018-05-20-0000/src/SimG4Core/CustomPhysics/src/FullModelHadronicProcess.cc 
>> Compiling  /build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_9_3_X_2018-05-20-0000/src/SimG4Core/CustomPhysics/src/CMSDarkPairProduction.cc 
In file included from /build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_9_3_X_2018-05-20-0000/src/SimG4Core/CustomPhysics/src/FullModelHadronicProcess.cc:6:0:
/build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_9_3_X_2018-05-20-0000/poison/SimG4Core/CustomPhysics/interface/G4ProcessHelper.hh:1:2: error: #error THIS FILE HAS BEEN REMOVED FROM THE PACKAGE.
 #error THIS FILE HAS BEEN REMOVED FROM THE PACKAGE.
  ^~~~~
>> Compiling  /build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_9_3_X_2018-05-20-0000/src/SimG4Core/CustomPhysics/src/CustomPhysics.cc 
/build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_9_3_X_2018-05-20-0000/src/SimG4Core/CustomPhysics/src/FullModelHadronicProcess.cc: In member function 'virtual G4bool FullModelHadronicProcess::IsApplicable(const G4ParticleDefinition&)':
/build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_9_3_X_2018-05-20-0000/src/SimG4Core/CustomPhysics/src/FullModelHadronicProcess.cc:23:19: error: invalid use of incomplete type 'class G4ProcessHelper'
</pre>

--- a/SimG4Core/CustomPhysics/src/FullModelHadronicProcess.cc
+++ b/SimG4Core/CustomPhysics/src/FullModelHadronicProcess.cc
@@ -3,7 +3,7 @@
 #include "G4ParticleTable.hh"
 
 #include "SimG4Core/CustomPhysics/interface/FullModelHadronicProcess.hh"
-#include "SimG4Core/CustomPhysics/interface/G4ProcessHelper.hh"
+#include "SimG4Core/CustomPhysics/interface/G4ProcessHelper.h"
 #include "SimG4Core/CustomPhysics/interface/Decay3Body.h"
 #include "SimG4Core/CustomPhysics/interface/CustomPDGParser.h"
 #include "SimG4Core/CustomPhysics/interface/CustomParticle.h"

